site/pages/retirement/[beneficiary_address]/[retirement_index].tsx

```diff
@@ -0,0 +1,57 @@
+import { GetStaticProps } from "next";
+
+import { getRetirementIndexInfo } from "@klimadao/lib/utils";
+import { SingleRetirementPage } from "components/pages/Retirement/SingleRetirement";
+import { loadTranslation } from "lib/i18n";
+import { IS_PRODUCTION } from "lib/constants";
+
+export const getStaticProps: GetStaticProps = async (ctx) => {
+  try {
+    if (IS_PRODUCTION) {
+      throw new Error("Not on Staging");
```

Comment:
return `props.notFound` is also an option

Reply:
I thought to keep the string as we can later see on the Vercel logs the reason why the page was not generated / resulted in 404.